scripts/home.js

```diff
@@ -66,27 +64,21 @@ createJsonRequest( method, url, function( err, response ) {
         if (contentText.length > response.textVisibleLength){
             parentDivElement.appendChild(buttonReadMore);
         }
-        
-        /* ================================================================================= */
-        /*
-            <!--Make dynamic-->
-            <thead >
-                <tr>
-                    <th>Lorem</th>
-                    <th>Lorem</th>
-                    <th>Lorem</th>
-                    <th>Lorem</th>
-                </tr>
-            </thead>
-        */
-        const tableHead = document.querySelector("table thead");
-        const tableRow = document.createElement("tr");
-        response.tableHeaders.forEach(element => {
-            const rowHead = document.createElement("th");
-            rowHead.innerText = element; 
-            tableRow.appendChild(rowHead);
-        });
-        tableHead.appendChild(tableRow);
+
+        // Table Area heading
+        const tableHead = document.querySelector(".table__heading");
+        tableHead.innerText = response.tableHeading;
+        parentDivElement.appendChild(tableHead);
     }
 });
 
+
+
+const tableHeaderUrl = "http://127.0.0.1:5500/config/tableHome_Header.json",
```

Comment:
Make a config file to keep all the base URLs instead of adding it many places like here

Reply:
Restructured URL as suggested.